### ISSUE TITLE
Locale not honored in Github Actions

### ISSUE THREAD

**Issue by natesulat** [milesj/boost#84]
Reference repo: https://github.com/natesulat/SampleRepo

Working on a project at work where I'm using `@boost/translate` to generate a static map of translations. The locale is passed to the NodeJS script using a Shell argument.

Locally, it works as expected, but when using Github Actions, the behavior changes.

**Implementation**
JS: https://github.com/natesulat/SampleRepo/blob/master/Demo.js
Shell: https://github.com/natesulat/SampleRepo/blob/master/show-the-issue.sh
Translation files: https://github.com/natesulat/SampleRepo/tree/master/language

**Expected**
This is what happens locally
```
sh ./show-the-issue.sh en-US
Locale is en-US
{"KEY":"favor","ONLY_EN":"hello","ONLY_US":"howdy","OVERRIDE":"LEET"}
sh ./show-the-issue.sh
Locale is en
{"KEY":"favour","ONLY_EN":"hello","ONLY_US":"onlyUS","OVERRIDE":"LEET"}
sh ./show-the-issue.sh dev
Locale is dev
{"KEY":"F4V0|2","ONLY_EN":"onlyEN","ONLY_US":"onlyUS","OVERRIDE":"1337"}
```

**Actual**
In Github Actions, there is no difference in output if the locale input is `en-US` or `en`
https://github.com/natesulat/SampleRepo/runs/669497498?check_suite_focus=true#step:5:1

Not sure if this is an issue with `i18next` or not, but this is the library we're using as a dependency in our project, so figured I'd come here first to ask for help.

Let me know if I can provide anymore info!

**Comment by milesj** [comment]
@natesulat This is odd. Can you set this and send me the debug output? `DEBUG=boost:*`

**Comment by natesulat** [comment]
No problem. Here's the output: https://github.com/natesulat/SampleRepo/runs/799968692?check_suite_focus=true

I set the variable in my shell script rather than defining it in in the Github Actions config, but let me know if you want me to do that and I can update the implementation.

**Comment by milesj** [comment]
@natesulat Been debugging a lot to figure this out and I think it's a case sensitivity issue. In my docs I suggest the format of `en-US` for the folder name, but it looks like `i18next` lower cases the locale to `en-us`. This works on my OSX machine, but not on linux. 

Don't think I can fix this as it's upstream, but can you try with a lower cased folder name?

**Comment by milesj** [comment]
PR fix: https://github.com/milesj/boost/pull/99 Will be part of v2.

**Comment by natesulat** [comment]
Awesome! Thank you for looking into and fixing. I've updated my sample repo to use 2.0.0-alpha.1 and the change seems to have worked, so much appreciated. Apologies if I'm overstepping here, but I'm going to close out this issue so it's not on your plate anymore.

What's your timeline for publishing v2 as the stable version? I'm in no rush to install it, because we were able to circumvent the problem, but I'd love to discard our hacky workaround at some point in favor of your version.
